packages/patternfly-react/src/components/VerticalNav/VerticalNav.test.js

```diff
@@ -42,14 +42,28 @@ test('VerticalNav renders properly with a custom className on a nav item', () =>
 test('VerticalNav renders properly with item objects', () => {
   const component = mount(
     <VerticalNav items={mockNavItems} pinnableMenus>
-      <Masthead>
-        <Brand
-          iconImg="../../../storybook/img/logo-alt.svg"
-          titleImg="../../../storybook/img/brand-alt.svg"
-        />
-        <IconBar>
-          <MockIconBarChildren />
-        </IconBar>
+      <Masthead title="Patternfly React">
+        <Masthead.Collapse>
```

Comment:
All of this is not needed.  You will only need the top `<Masthead />` to verify that the correct component is rendered.  This just confuses the test, and makes it seem like it is doing more. This can even be made smaller once masthead is switched to a prop.